src/stackdriver-nozzle/nozzle/handler.go

```diff
@@ -0,0 +1,7 @@
+package nozzle
+
+import "github.com/cloudfoundry/sonde-go/events"
+
+type Handler interface {
```

Comment:
suggest a different naming strategy, make the interface name specific and the Handle function generic (because the type is already specified in the param type)